toonz/sources/toonz/shortcutpopup.cpp

```diff
@@ -191,7 +191,7 @@ ShortcutTree::ShortcutTree(QWidget *parent) : QTreeWidget(parent) {
   addFolder(tr("Fill"), FillCommandType);
   addFolder(tr("File"), MenuFileCommandType, menuCommandFolder);
   addFolder(tr("Edit"), MenuEditCommandType, menuCommandFolder);
-  addFolder(tr("Scan & Cleanup"), MenuScanCleanupCommandType,
+  addFolder(tr("Scan && Cleanup"), MenuScanCleanupCommandType,
```

Comment:
It would be better to split this change to a separated PR, since it is not directly related to Chinese UI localisation. 
@skitaoka